src/js/app.js

```diff
@@ -9,6 +9,7 @@ myApp.config(function($routeProvider, $httpProvider, $translateProvider, $compil
   $translateProvider.preferredLanguage('cn');
   $translateProvider.useSanitizeValueStrategy('escape');
 
+  $compileProvider.aHrefSanitizationWhitelist(/^\s*(http|https|javascript)/);
```

Comment:
This fixes an issue with other links as well, where angular is adding "unsafe" to the front of them.

If we want to be extra-cautious, we would need to add a regex specifically for that javascript:require('electron') bit